advanced/Templates/pihole-FTL.service

```
@@ -47,7 +76,7 @@ start() {
 # Stop the service
 stop() {
   if is_running; then
-    pkill -xo "pihole-FTL"
+    pkill -F "${FTL_PID_FILE}"
```

Use more generic/low level `kill` command:
```suggestion
    kill "$FTL_PID"
```